plugins/orchestrator-common/src/auto-generated/openapi-client/.openapi-generator-ignore

```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
```

Comment:
If these are generated files, why commit them to the repo?  Would it be better to generated this as a post-install thing perhaps?

Reply:
we generate only when needed. upon demand.